SDPhysicsEngine/Sources/SDPhysicsEngine/Object/SDCameraObject.swift

```diff
@@ -2,9 +2,29 @@ import SpriteKit
 
 public class SDCameraObject: SDObject {
     let cameraNode: SKCameraNode
+    let player: SDObject?
 
     override public init() {
+        player = nil
         cameraNode = SKCameraNode()
         super.init(node: cameraNode)
     }
+
+    init(player: SDObject) {
+        self.player = player
+        cameraNode = SKCameraNode()
+        super.init(node: cameraNode)
+    }
+
+    var zRotation: CGFloat {
+        get { cameraNode.zRotation }
+        set { cameraNode.zRotation = newValue }
+    }
+
+    func update() {
+        guard let player = self.player else {
+            return
+        }
+        self.position = player.position
```

Comment:
I noticed it was previously only tracking the x position, but the player may jump up to higher platforms and exit the height of what the view can see...

Reply:
Hmm that's true. My rationale for fixing the camera's y position was because if the camera track's the player's y position as well, then we may run out of the background image when going up. I guess it can be said that levels should not be limited in height (maybe can have a level that's focused in the sky or something), but with the lack of assets, would it be better to just cap how high the player can go?

Reply:
Maybe we resolve this after we have a complete level ready? We can test then what feels most normal/natural. We can create an issue in the meantime

Reply:
okay that sounds good!